.github/labeler.yml

```diff
@@ -1,106 +1,155 @@
 # list of labels and minimatch globs to match to apply the label.
 
 CI:
-- any: ['.cirrus.yml']
-- any: ['.github/dependabot.yml']
-- any: ['.github/labeler.yml']
-- any: ['.github/workflows/*']
-- any: ['.appveyor.yml']
-- any: ['.codecov.yml']
+  - all:
+      - changed-files:
+          - any-glob-to-all-files:
+              - '.cirrus.yml'
+              - '.github/dependabot.yml'
+              - '.github/labeler.yml'
+              - '.github/workflows/*'
+              - '.appveyor.yml'
+              - '.codecov.yml'
 
 documentation:
-- all: ['*/doc/*']
+  - all:
+      - changed-files:
+          - any-glob-to-all-files:
+              - '*/doc/*'
 
 translation:
-- any: ['src/po/*.po']
+  - all:
+      - changed-files:
+          - any-glob-to-all-files:
+              - 'src/po/*.po'
```

Comment:
With the default settings of Vim, the indents become different:
```diff
--- a/.github/labeler.yml
+++ b/.github/labeler.yml
@@ -19,9 +19,9 @@ documentation:
 
 translation:
   - all:
-      - changed-files:
-          - any-glob-to-all-files:
-              - 'src/po/*.po'
+    - changed-files:
+      - any-glob-to-all-files:
+        - 'src/po/*.po'
 
 if_lua:
   - all:
```

Maybe it is better to follow the default setting of Vim. (Or add modelines?)